### PR TITLE
Fix modified gpgkey init

### PR DESCRIPTION
**Pull request by JacobCallahan**
Elements of this modification should have remained instances of self

### CODE REVIEW

File: robottelo/config/__init__.py [hunk 1]
```diff
@@ -133,8 +133,8 @@ def configure_nailgun():
 
     def patched_gpgkey_init(self, server_config=None, **kwargs):
         """Set a default value on the ``content`` field."""
-        gpgkey_init(settings, server_config, **kwargs)
-        settings._fields['content'].default = str(
+        gpgkey_init(self, server_config, **kwargs)
+        self._fields['content'].default = str(
```
Comment (mshriver) [inline robottelo/config/__init__.py:137]: Shouldn't this be setting `_fields` on the GPGKey entity instance returned by calling `entities.GPGKey.__init__`? The `gpgkey_init()` return needs to be captured and referenced here?

Comment (mshriver) [reply robottelo/config/__init__.py:137]: Taking a second look, `self` is just _VERY_ misleading in this context.

Comment (JacobCallahan) [reply robottelo/config/__init__.py:137]: discussed in chat

Comment (mshriver) [reply robottelo/config/__init__.py:137]: The ugliness of this baby won't be fully addressed here.

**Review by mshriver — APPROVED** [review]

**Pull request merged**